pkgs/top-level/all-packages.nix

```diff
@@ -39266,7 +39266,7 @@ with pkgs;
   dapper = callPackage ../development/tools/dapper { };
 
   kube3d =  callPackage ../applications/networking/cluster/kube3d {
-    buildGoModule = buildGo118Module; # tests fail with 1.19
+    buildGoModule = buildGo120Module;
```

Comment:
Thats the default, so we could remove it

Reply:
It wasn't at the time, thanks for raising

```
λ git branch | grep '*'
* kube3d-update
λ rg "^  buildGoModule ="
pkgs/top-level/all-packages.nix
24242:  buildGoModule = buildGo119Module;
λ git checkout master
Switched to branch 'master'
Your branch is up to date with 'origin/master'.
λ rg "^  buildGoModule ="
pkgs/top-level/all-packages.nix
24381:  buildGoModule = buildGo120Module;
```